src/Hamiltonians/HubbardRealSpace.jl

```diff
@@ -113,15 +113,50 @@ It is implemented recursively to ensure type stability.
     return self + row + _interactions(as, rest)
 end
 
+"""
+    trap_potential(add::SingleComponentFockAddress, pot::Array)
+
+Calculate trap potential energy of a single-component address `add`. The (precomputed) 
+potential energy per particle at each point in the lattice is given by the array `pot`.
+```math
+V = \\sum_{d=1}^D \\sum_{i=1}^M V_d x_{i,d}^2 n_i,
+```
+where ``x = (x_1, ..., x_D)`` is the distance of site `i` to the centre of the trap and 
+`n_i` is the number of particles at site `i`. `M` is the total number of sites on the 
+lattice.
+"""
+function trap_potential(add::SingleComponentFockAddress, pot::Array)
+    pe = 0.
```

Comment:
Style comment: It's usually preferred to spell it out, e.g. `0.0`. I don't feel *too* strongly about that though.

Reply:
oops old Fortran habits...